src/Connections/GenericHook.ts

```diff
@@ -355,10 +393,10 @@ export class GenericHookConnection extends BaseConnection implements IConnection
      * @returns `true` if the webhook completed, or `false` if it failed to complete
      */
     public async onGenericHook(data: unknown): Promise<boolean> {
-        log.info(`onGenericHook ${this.roomId} ${this.hookId}`);
+        log.info(`onGenericHook ${this.roomId} ${this.state.hookId}`);
```

Comment:
What's the reason for changing this?

Reply:
It was to deduplicate the `hookId` from being both a property of `GenericHookConnection` and its `state` object, since in practice the `hookId` is always set to the `state`'s `name`.

Reply:
Reverted